### ISSUE TITLE
No suggested usings on property methods

### ISSUE THREAD

**Issue by miloush** [dotnet/roslyn#23671]
**Version Used**: 15.5.0 Preview 5

**Steps to Reproduce**:
```C#
public int MyProperty
{
    [MethodImpl|
    get { }
}
```

**Expected Behavior**: Add "using System.Runtime.CompilerServices;" suggested. Note that automatic brace completion is off and the attribute does not have the closing bracket yet.

**Actual Behavior**: No suggestions. The same scenario works on methods though.

**Comment by dpoeschl** [comment]
Like you implied, it works with brace completion active (or the close bracket typed). @kuhlenh do we know how many devs disable brace completion? Just curious.

**Comment by Neme12** [comment]
Note: this is only a problem because the compiler doesn't report these errors on unknown accessors.
You don't get the code fix here either:
```c#
class C
{
    int P
    {
        [STAThread]
    }
}
```
or here:
```c#
class C
{
    int P
    {
        [STAThread]
        what { }
    }
}
```

It seems to me like errors should be reported here given that they are reported on incomplete member as well?

**Comment by CyrusNajmabadi** [comment]
@gafter For another case of binding of attributes on incomplete members (this time accessors though).
